packages/buidler-core/src/internal/util/scripts-runner.ts

```diff
@@ -35,6 +43,8 @@ export async function runScriptWithBuidler(
   extraNodeArgs: string[] = [],
   extraEnvVars: { [name: string]: string } = {}
 ): Promise<number> {
+  log("Creating Buidler subprocess to run " + scriptPath);
```

Comment:
We should probably stick to using backticks:
```js
log(`Creating Buidler subprocess to run ${scriptPath}`);
```

Reply:
We can enable https://palantir.github.io/tslint/rules/prefer-template/ in a separate PR